plugins/fs/src/lib.rs

```diff
@@ -48,25 +43,6 @@ impl<R: Runtime, T: Manager<R>> FsExt<R> for T {
     }
 }
 
-impl ScopeObject for scope::Entry {
-    type Error = Error;
-    fn deserialize<R: Runtime>(
-        app: &AppHandle<R>,
-        raw: Value,
-    ) -> std::result::Result<Self, Self::Error> {
-        #[derive(Deserialize)]
-        struct EntryRaw {
-            path: PathBuf,
-        }
-
-        let entry = serde_json::from_value::<EntryRaw>(raw.into())?;
-
-        Ok(Self {
-            path: app.path().parse(entry.path)?,
```

Comment:
you removed this bit :) gonna push it again

Reply:
thanks, wasn't sure why deserialization was done this way and didn't notice the resolve part.